nf_core/pipeline-template/nextflow.config

```diff
@@ -58,6 +56,13 @@ params {
     max_cpus                   = 16
     max_time                   = '240.h'
 
+    // Schema validation default options
+    validationFailUnrecognisedParams = false
+    validationLenientMode            = false
+    validate_params                  = true
```

Comment:
Should we also make them camel-case?

Reply:
I'm doubting with `validate_params`. For `schema_ignore_params ` you are right, it should be camelCase as it's part of the nf-validation plugin. But `validate_params` is only used in nf-core pipelines, so I think it should be snake_case?

Reply:
I'd go for all `snake_case`

Reply:
if it's params, then `snake_case`

Reply:
Unfortunately the nf-validation plugin is using camelCase as nextflow uses this notation, and it's ment to be for all NF pipelines and not only nf-core.

Reply:
Good point, then ok for camelCase